utopia-remix/app/routes/projects.tsx

```diff
@@ -804,7 +804,7 @@ const ProjectCard = React.memo(
             </div>
           </div>
         </ContextMenu.Trigger>
-        <ProjectActionsMenu project={project} accessRequests={accessRequests} />
+        <ProjectActionsMenu project={project} />
         <SharingDialogWrapper project={project} accessRequests={accessRequests} />
```

Comment:
can we also move the `accessRequestFetcher` login inside `SharingDialogWrapper` (since now, after the change, it's the only place that uses it)

Reply:
yep, coming in the next PR which does a larger set of improvements (including this one!)